src/utils.ts

```diff
@@ -1,14 +1,25 @@
 import { build, Plugin } from 'vite';
 import { OutputChunk } from 'rollup';
 
+export type InjectCode = (cssCode: string, styleId?: string) => string;
+
 const cssInjectedByJsId = '\0vite/all-css';
 
-export async function buildCSSInjectionCode(cssToInject: string, styleId?: string): Promise<OutputChunk | null> {
+const defaultInjectCode: InjectCode = (cssCode, styleId) =>
+    `try{if(typeof document != 'undefined'){var elementStyle = document.createElement('style');${
+        typeof styleId == 'string' && styleId.length > 0 ? `elementStyle.id = '${styleId}';` : ''
+    }elementStyle.appendChild(document.createTextNode(${cssCode}));document.head.appendChild(elementStyle);}}catch(e){console.error('vite-plugin-css-injected-by-js', e);}`;
+
+export async function buildCSSInjectionCode(
+    injectCode: InjectCode = defaultInjectCode,
+    cssToInject: string,
+    styleId?: string
+): Promise<OutputChunk | null> {
```

Comment:
Can you move `injectCode` param after `styleId`?

Reply:
Do you mean only here or in all other places it's declared?

---

I was trying to be consistent and have it in the same position across different functions, but I can't move it to after `styleId` in other places, since `styleId` is optional. For example:

![image](https://user-images.githubusercontent.com/25781956/203804325-e6e69d9f-5634-4add-b0dd-a873a3b232a0.png)